charts/csi-secrets-store-provider-azure/templates/provider-azure-installer.yaml

```diff
@@ -22,13 +22,17 @@ spec:
 {{- if .Values.linux.podAnnotations}}
 {{- toYaml .Values.linux.podAnnotations | nindent 8 }}
 {{- end }}
-{{- if and .Values.enableArcExtension .Values.arc.enableMonitoring }}
```

Comment:
`and` didn't work here?

Reply:
nope. It tries to evaluate `.Values.arc.enableMonitoring` as well which is not in values.yaml